torch/_inductor/codegen/cuda/cutlass_utils.py

```diff
@@ -16,6 +16,10 @@
 
 log = logging.getLogger(__name__)
 
+# This is a feature flag. If _DISABLE_CUTLASS_BACKEND is set to true,
+# then the CUDA / CUTLASS backend is entirely disabled, including it's unit tests
+_DISABLE_CUTLASS_BACKEND = False
```

Comment:
lets use `torch._inductor.config` for this.

Reply:
I don't think we should do that, for the following reasons:

 * This is not intended as config setting that may be changed. 
 * It's a constant to disable experimental or partially implemented features so I can split the PR into multiple parts.
 * If I would make this a config option, the expectation would be that it's permissible to change it.
 * It would become part of the public interface, so I can't just remove it later without potentially.

Reply:
I agree that the flag is confusing. My impression is that the main purpose of having this flag is splitting the PRs in this stack? How much sense does this flag make after you PRs are landed? Are you going to remove this flag in one of the PRs in the stack, as it's no longer relevant? If so, maybe simply disable some of the unit tests in the middle of the stack (when they are not supposed to pass) and enable back in a subsequent PR, to achieve what you want?

Reply:
You got that correctly, it's for splitting the PR. Yes, I can remove it afterwards if you prefer.  I would have to remove all references to it, though, which are quite a few, so I would create a separate PR for that.

Reply:
@jansel if the flag will be removed after all the PRs land, would it be acceptable to keep it around in-between?

Reply:
Let's actually split the PR into logical chunks (each with their own set of tests).

Splitting the tests from the code they are testing is kind of defeating the point of splitting up the PR.

I also want CI the be running on this code.  So we should have tests that patch this flag enabled and run it.